### DISCUSSION TITLE
New Installation 1 Abuntu 20.04 VM with Security Onion 2.4 and 6 Hardware Sensors

### DISCUSSION THREAD

**Discussion by Nazeer1982** [Security-Onion-Solutions/securityonion#11596] [2.4]
### Version

2.4.0

### Installation Method

Network installation on Ubuntu

### Description

installation

### Installation Type

Standalone

### Location

cloud

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

128

### Storage for /

1000

### Storage for /nsm

800

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Hi All,

I have one VM in Azure cloud and 6 Sensors.  I was wondering was it possible to setup on standalone in the cloud and add the sensors. I was hoping to setup the sensors in heavymode + sensor.  Is this possible? 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by cm-ops** [comment]
Just so I am clear, you want to setup a node in the cloud and have on-prem sensors send to it? Or everything in the cloud?
